components/KiboPersonalDetails.vue

```diff
@@ -0,0 +1,205 @@
+                  @input="updateField('firstName', $event)"
+                />
+
+                <SfInput
+                  :value="value.lastName"
+                  :label="inputsLabels[1]"
+                  name="lastName"
+                  class="form__element form__element--half"
+                  required
+                  @input="updateField('lastName', $event)"
+                />
+
+                <SfInput
+                  :has-show-password="true"
+                  type="value.password"
+                  :label="createAccountInputLabel"
+                  class="form__element"
+                  required
+                  data-testid="create-password-input"
+                />
+              </div>
+            </transition>
+          </slot>
+        </div>
+      </slot>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { SfInput, SfCheckbox, SfButton, SfHeading, SfCharacteristic } from "@storefront-ui/vue"
+
+export default {
+  name: "KiboPersonalDetails",
+  components: {
+    SfInput,
+    SfCheckbox,
+    SfButton,
+    SfHeading,
+    SfCharacteristic,
+  },
+  props: {
+    value: {
+      type: Object,
+      default: () => ({}),
+    },
+    buttonText: {
+      type: String,
+      default: "Log into your account",
+    },
+    logInInfo: {
+      type: String,
+      default: "or fill the details below:",
+    },
+    headingTitle: {
+      type: String,
+      default: "Personal details",
+    },
+    headingTitleLevel: {
+      type: Number,
+      default: 2,
+    },
+    inputsLabels: {
+      type: Array,
+      default: () => ["First name", "Last name", "Your email"],
+    },
+    additionalDetails: {
+      type: String,
+      default: "Enjoy these perks with your free account!",
+    },
+    characteristics: {
+      type: Array,
+      default: () => [
+        { description: "Faster checkout", icon: "clock", size: "24px" },
+        {
+          description: "Earn credits with every purchase",
+          icon: "credits",
+          size: "24px",
+        },
+        {
+          description: "Full rewards program benefits",
+          icon: "rewards",
+          size: "24px",
+        },
+        { description: "Manage your wishlist", icon: "heart", size: "24px" },
+      ],
+    },
+    transition: {
+      type: String,
+      default: "sf-fade",
+    },
+    createAccountCheckboxLabel: {
+      type: String,
+      default: "I want to create an account",
+    },
+    createAccountInputLabel: {
+      type: String,
+      default: "Create Password",
+    },
+  },
+  setup(props, context) {
+    const updateField = (fieldName, fieldValue) => {
+      const values = {
+        ...props.value,
+        [fieldName]: fieldValue,
+      }
+
+      context.emit("input", values)
+    }
+
+    return {
+      createAccount: false,
+      updateField,
+    }
+  },
+}
+</script>
+
+<style lang="scss" scoped>
+@import "~@storefront-ui/shared/styles/components/templates/SfPersonalDetails.scss";
```

Comment:
Check if this is needed ?

Reply:
I have verified but its require still we will check